.github/workflows/tpu-tests.yml

```diff
@@ -97,42 +95,32 @@ jobs:
     - name: Update script
       run: |
         import os
-        fname = f'tests/tests_{os.getenv("SCOPE")}/run_tpu_tests.sh'
+        fname = f'tests/tests_${{ matrix.pkg-name }}/run_tpu_tests.sh'
         with open(fname) as fo:
             data = fo.read()
         data = data.replace('{PYTORCH_VERSION}', os.environ["XLA_VER"])
         data = data.replace('{PR_NUMBER}', os.environ["PR_NUMBER"] or "master")
         data = data.replace('{SHA}', os.environ["SHA"])
-        data = data.replace('{RUNTIME}', os.environ["RUNTIME"])
+        data = data.replace('{RUNTIME}', '${{ matrix.runtime }}')
         print(data)
         with open(fname, "w") as fw:
             fw.write(data)
       shell: python
 
     - name: Create node
-      id: tpu-create
-      continue-on-error: true
       env:
         JOB_NAME: ${{ env.PR_NUMBER }}-${{ matrix.pkg-name }}-${{ matrix.runtime }}-${{ env.SHA }}
       run: |
         gcloud compute tpus tpu-vm create "$JOB_NAME" --accelerator-type=v4-8 --version="tpu-vm-v4-pt-$XLA_VER" --preemptible
 
-    - name: Cancel job
```

Comment:
Again?

Reply:
This is why I'm asking that you check https://github.com/Lightning-AI/lightning/pull/18052 again. This is very urgent